Pipfile

```diff
@@ -5,6 +5,7 @@ name = "pypi"
 
 [packages]
 flair = {editable = true, path = "."}
+tiny-tokenizer = {extras = ["all"], version = "*"}
```

Comment:
I think this line can be removed, it is enough to specify tiny_tokenizer as a dependency in `requirements.txt`.

Reply:
Removed in a945b62. 👍